alerting/src/main/kotlin/org/opensearch/alerting/DocumentLevelMonitorRunner.kt

```diff
@@ -83,7 +88,10 @@ object DocumentLevelMonitorRunner : MonitorRunner() {
         logger.debug("Document-level-monitor is running ...")
         val isTempMonitor = dryrun || monitor.id == Monitor.NO_ID
         var monitorResult = MonitorRunResult<DocumentLevelTriggerRunResult>(monitor.name, periodStart, periodEnd)
-
+        var nonPercolateSearchesTimeTaken = AtomicLong(0)
+        var percolateQueriesTimeTaken = AtomicLong(0)
+        var totalDocsQueried = AtomicLong(0)
+        var docTransformTimeTaken = AtomicLong(0)
```

Comment:
why do we need these variables to be `atomiclong` because `DocumentLevelMonitorRunner` runs in a single thread?

Reply:
Agreed that this is a single threaded operation but `AtomicLong.getAndAdd()` updates the final variable while Kotlin `Long` is immutable and would require me to use new variables if i do additions or any math operations .
```
val docsSizeInBytes: Long = 0L
val updatedDocsSizeInBytes = docsSizeInBytes + 5
```
I was trying to avoid initializing new variables.

Reply:
we can change `val` to `var` & it becomes mutable

Reply:
using LongAdder instead of var as they're being passed to different methods

Reply:
why can't we do something like this?
```
var docsSizeInBytes: Long = 0L
docSizeinBytes = docSizeInBytes.plus(5)
```